version.c

```diff
@@ -90,215 +89,243 @@ static const char *Notice =
 struct compile_options
 {
   const char *name;
-  int enabled;
+  bool enabled;
 };
 
+/* These are sorted by the display string */
 static struct compile_options comp_opts[] = {
-#ifdef CRYPT_BACKEND_CLASSIC_PGP
-  { "CRYPT_BACKEND_CLASSIC_PGP", 1 },
-#else
-  { "CRYPT_BACKEND_CLASSIC_PGP", 0 },
-#endif
-#ifdef CRYPT_BACKEND_CLASSIC_SMIME
-  { "CRYPT_BACKEND_CLASSIC_SMIME", 1 },
-#else
-  { "CRYPT_BACKEND_CLASSIC_SMIME", 0 },
-#endif
-#ifdef CRYPT_BACKEND_GPGME
-  { "CRYPT_BACKEND_GPGME", 1 },
+  { "attach_headers_color", 1 },
+#ifdef HAVE_BKGDSET
+  { "bkgdset", 1 },
 #else
-  { "CRYPT_BACKEND_GPGME", 0 },
+  { "bkgdset", 0 },
 #endif
-#ifdef DEBUG
-  { "DEBUG", 1 },
+#ifdef HAVE_COLOR
+  { "color", 1 },
 #else
-  { "DEBUG", 0 },
+  { "color", 0 },
 #endif
-#ifdef DL_STANDALONE
-  { "DL_STANDALONE", 1 },
+  { "compose_to_sender", 1 },
+  { "compress", 1 },
+#ifdef USE_COMPRESSED
+  { "compressed", 1 },
```

Comment:
Ah, you got me.  That's what happens when I rely on "sort | uniq -d" rather than checking by eye.

Stripping out the PATCHES **and** the unconditionals leaves:
```
Compile options:
  +bkgdset +color +crypt_pgp +crypt_smime +crypt_gpgme +curs_set +debug 
  +dl_standalone -dotlock +enable_nls -exact_address +fcntl +flock -fmemopen 
  +futimens +getaddrinfo +gss -homespool -iconv_nontrans +libidn -locales_hack 
  +lua +meta +mixmaster +notmuch +resizeterm +sasl +setgid +ssl_gnutls 
  -ssl_openssl +start_color -sun_attachment +typeahead +wc_funcs
```